src/10_functions.py

```diff
@@ -2,11 +2,24 @@
 
 # YOUR CODE HERE
 
+
+def is_even(num):
+    if num % 2 == 0:
+        return True
+    else:
+        return False
+
+
+print(is_even(2))
+
 # Read a number from the keyboard
 num = input("Enter a number: ")
 num = int(num)
 
 # Print out "Even!" if the number is even. Otherwise print "Odd"
 
 # YOUR CODE HERE
-
+if num % 2 == 0:
+    print("Even!")
+else:
+    print("Odd!")
```

Comment:
As we discussed, this second if else is not necessary. You can just call the method is_even.